<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信号槽的5种链接方式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_max">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>最大化显示（1、SINAL/SLOT）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_normal">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>正常显示（2、函数地址）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_min">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>最小化显示（3、设计师界面-转到槽）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>关闭窗口（4、设计师界面-信号槽编辑器）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_set">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改窗口标题（5、lambda表达式）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
